openshift/e2e-common.sh

```diff
@@ -191,6 +191,29 @@ function run_e2e_tests(){
   local test_name=$1 
   local failed=0
 
+  # Keep this in sync with test/ha/ha.go
+  readonly REPLICAS=2
+  readonly BUCKETS=10
+
+  # Keep the bucket count in sync with test/ha/ha.go
+  # TODO: configure it in KnativeServing when operator supports it.
+  kubectl -n "${SYSTEM_NAMESPACE}" patch configmap/config-leader-election --type=merge \
+    --patch='{"data":{"buckets": "'${BUCKETS}'"}}' || failed=1
+
+  # Changing the bucket count and cycling the controllers will leave around stale
+  # lease resources at the old sharding factor, so clean these up.
+  kubectl -n ${SYSTEM_NAMESPACE} delete leases --all
+
+  # Wait for a new leader Controller to prevent race conditions during service reconciliation
+  wait_for_leader_controller || failed=1
```

Comment:
I'm surprised that the Leases will pop up again when you've just deleted them, without any manipulation with pods. But ok:)